### DISCUSSION TITLE
更新日志 - 20230320

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#650] [Announcements]
# 系统：
1. 修复 LUCI 防火墙状态的规则链接在使用默认的 Argon 主题时无效的问题 #148
2. CDC 网络驱动支持获取速率 （不包括 RK3568 机型）
3. 修复 5.4 内核安装 OpenWRT 官方内核模块可能无效的问题 #621

# 应用：
* QuickStart：
  1. 修复首页显示的 IPv6 地址不是实际出口地址的问题 #631

# 设备：
* R4S：增加 PWM 风扇配置 #271
* R4S/R2S：修改 USB 的配置，可能增加稳定性和性能，其中 R2S 同时还影响 LAN 口 #638

**Comment by jjm2473** [comment]
CDC 网络驱动是用来给未提供驱动的 USB 网卡，作为通用驱动。

**Reply by phcl1998** [reply]
https://s2.loli.net/2023/03/24/mXqpCSPGKVjeZtU.png

**Reply by phcl1998** [reply]
> https://s2.loli.net/2023/03/24/mXqpCSPGKVjeZtU.png
q群网友反馈的，恢复备份少了一个字

**Comment by 0xJacky** [comment]
您好，R5S 点击检查更新还是提示 2023031010 是最新版本

**Reply by jjm2473** [reply]
只是编译了，还没发布到服务器上，等小宝有空了操作发布流程。一般都会延迟一天

**Reply by 0xJacky** [reply]
好的谢谢

**Comment by mashir0o0** [comment]
r68s线刷这版固件后无法开机 退回r68s-update-0610 最初始版本后可以刷入（其他固件都行；Openwrt Etc），但是img.gz的Istoreos固件（新方法刷固件）刷入后都无法开机。

**Reply by jinxin85** [reply]
我是虚拟机,下载的 istoreos-21.02.3-2023031713-x86-64-squashfs-combined.img.gz 包, 因在线升级太慢, 我下载后直接更新系统, 然后无限重启, 这个时候才想起来文件校验,校验失败,重新下载后 依然校验失败,等更新吧

**Reply by jjm2473** [reply]
@mashir0o0 你是不是没解压，用线刷工具要先解压

**Reply by mashir0o0** [reply]
@jjm2473 是需要把img.gz解压吗 因为网站上下载下来就是img.gz了

**Reply by jinxin85** [reply]
我下载的gz包 有问题, 下载很多次都是文件损坏

**Reply by jjm2473** [reply]
> @jjm2473 是需要把img.gz解压吗 因为网站上下载下来就是img.gz了

对，要解压成img。如果使用命令的话，可以用gunzip命令

**Reply by jjm2473** [reply]
> 我下载的gz包 有问题, 下载很多次都是文件损坏

应该是上传出了问题，等明天重发版本吧。

**Reply by mashir0o0** [reply]
@jjm2473 解压成img 在用线刷工具手动进maskrom刷是吗 我去试试谢谢

**Reply by mashir0o0** [reply]
@jjm2473 解决了 谢谢！官方教程和我之前刷机经验都是直接刷img.gz的，我刚才也尝试过解压后但是放在固件升级里但是显示固件错误就没再尝试了

**Reply by jjm2473** [reply]
@mashir0o0 后台网页升级不需要手动解压。usb刷机才需要手动解压，因为那是rockchip的工具，它只会当成数据直接写emmc。

**Comment by jjm2473** [comment]
部分固件上传错误，手动下载请检查hash值再刷机。明天会重新发个版本。

**Reply by jjm2473** [reply]
已重新发版

**Comment by nevertoday** [comment]
今天这个固件，慢如蜗牛~ 两个小时都还没有更新下来，请问，我要如何操作？？
平时几分钟的事情~

**Reply by jinxin85** [reply]
去固件服务器自行下载, 下载工具可以用idm 迅雷,一定要校验 ,反正我下载的文件是坏的

**Reply by weiyideai520** [reply]
以前下载很快的，也就是最近一两个估计非常的慢。

**Reply by jjm2473** [reply]
这段时间限流了，之后我们搞好多IP加速就会快很多了

**Comment by qzwa** [comment]
系统的在线设备和实际不一样

**Reply by jjm2473** [reply]
在线设备实际是根据arp记录来的，如果设备都没用跟istoreos通信，那就不会计数

**Comment by qzwa** [comment]
在线更新都是失败

**Comment by nevertoday** [comment]
![image](https://user-images.githubusercontent.com/166048/226229253-3a83d928-fd41-413d-bdd6-6a2ac53110ab.png)
同款报错！

**Reply by jjm2473** [reply]
固件还在上传

**Reply by nevertoday** [reply]
这几天下载更新，真的好慢哈~ 感觉被按住脑袋了一般。
and
thanks for u reply!

**Reply by jjm2473** [reply]
这段时间限流了，之后我们搞好多IP加速就会快很多了

**Comment by jjm2473** [comment]
20230317上传错误，重新发布 20230320

**Comment by CXK558** [comment]
小宝你的固件安装TUROBO ACC IPK插件总是出错，请问如何处理
Unknown package 'luci-app-turboacc'.
Collected errors:
 * pkg_hash_check_unresolved: cannot find dependency kmod-tcp-bbr for luci-app-turboacc
 * pkg_hash_check_unresolved: cannot find dependency kmod-shortcut-fe-cm for luci-app-turboacc
 * pkg_hash_fetch_best_installation_candidate: Packages for luci-app-turboacc found, but incompatible with the architectures configured
 * opkg_install_cmd: Cannot install package luci-app-turboacc.

**Reply by jjm2473** [reply]
不支持

**Comment by qzwa** [comment]
要重新下载吗

**Reply by jjm2473** [reply]
rk3568方案的没什么更新，不需要升级，其他机型可以升级，增加了对openwrt官方内核模块的兼容性。

**Comment by ZJUprince** [comment]
请问升级后首页沙盒模式消失 如何解决呢
![image](https://user-images.githubusercontent.com/38859427/227565299-935dca72-dfc1-42cd-8dad-a517656f3cd3.png)

**Reply by jjm2473** [reply]
什么设备？最近没改沙箱的东西，是不是你删掉了沙箱的脚本？

**Reply by ZJUprince** [reply]
> 什么设备？最近没改沙箱的东西，是不是你删掉了沙箱的脚本？

天宝NAS，已经重置系统就有了，可能是我误删？
